packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -124,6 +138,19 @@ Each LogEvent object describes a single logged action or "event" that is perform
      "device": String, Optional
      "id": String, Optional
 },
+"device": { Object, Optional
+            "id": String, Optional
```

Comment:
Indent on this object fields is bigger, than on the rest of the fields.

![image](https://github.com/okta/okta-developer-docs/assets/32918538/54059ebb-222e-4370-80ac-e497bac21be0)